source/addComment.js

```diff
@@ -1,3 +1,27 @@
 $(document).ready(function(){
   // add your code here
+  function formChecker(comment, author) {
+    if(comment == "") {
+      return false
+    } else {
+      return true
+    };
+  };
+
+  $('#new_comment_button').click(function() {
+    $('.form').show();
+  })
+
+  $('#comment_editor').submit(function(){
+    let comment = $('#comment_editor :input')[0].value
```

Comment:
you could be more specific with your selector here so you don't have to hardcode the index here 👍